sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3163,9 +3190,10 @@ SUBROUTINE MISCLN
          IF ( (IGET(109)>0).OR.(IGET(110)>0) ) THEN
             CALL CALLCL(P1D,T1D,Q1D,EGRID1,EGRID2)
             IF (IGET(109)>0) THEN
+               GRID1=spval
 	       DO J=JSTA,JEND
                DO I=1,IM
-                 GRID1(I,J)=EGRID2(I,J)
+                 IF(T1D(I,J) < spval) GRID1(I,J)=EGRID2(I,J)
```

Comment:
suggest changing to:

 IF(EGRID2(I,J) < spval) GRID1(I,J)=EGRID2(I,J)

Reply:
will do

Reply:
done